scapy/contrib/gtp.py

```diff
@@ -192,14 +192,15 @@ def answers(self, other):
         return (isinstance(other, GTPHeader) and
                 self.version == other.version and
                 self.payload.answers(other.payload))
-
-    @classmethod
-    def dispatch_hook(cls, _pkt=None, *args, **kargs):
-        if _pkt and len(_pkt) >= 1:
-            if (struct.unpack("B", _pkt[0])[0] >> 5) & 0x7 == 2:
-                import gtp_v2
-                return gtp_v2.GTPHeader
-        return GTPHeader
+    

```

Comment:
Hey, I didn't change the code on the @classmethod, I only altered the IE_EndUserAddress() part, but when I saved the file that ^M which appeared. I edited it several times now trying to remove it, but was not able. Not in pycharm, vim, nano or even Sublime Text. I'm out of ideas on how to remove that ^M from the @classmethod.

All in all, I added more descriptions to the gtp.uts added tests, I've put the lambda expressions on the same line, and finally added RandIP6() to the IPv6 field.

Thanks for you feedback.